app/tests/settings.py

```diff
@@ -58,4 +58,7 @@
 # Ensure custom models for testing are found
 INSTALLED_APPS += [  # noqa F405
     "tests.core_tests",
+    "allauth.socialaccount.providers.dummy",
```

Comment:
Something for a separate PR - we should check that this provider is not installed in production by using a check (see `check_sagemaker_is_used`). I think that this provider is really dangerous, it allows you to log in as any user by just providing their email address, right?